driver-core/src/main/java/com/datastax/driver/core/querybuilder/BuiltStatement.java

```diff
@@ -297,14 +296,16 @@ public Boolean isIdempotent() {
     @Override
     public String toString() {
         try {
-            if (forceNoValues) {
-                return getQueryString();
-            }
-            StringBuilder queryString = buildQueryString(null, CodecRegistry.DEFAULT_INSTANCE);
-            return maybeAddSemicolon(queryString).toString();
+            // try first with all values inlined (will not work if some values require custom codecs)
```

Comment:
I see so it should behave the same but without using the cache.  I'm not completely sure i understand the motivation behind the change though.  What is the difference, that the values are included in the toString?